src/app/pages/view-test-result/components/test-summary-card/test-summary-card.html

```diff
@@ -24,25 +24,32 @@ <h4 id="view-test-result-test-summary-title">Test summary</h4>
                     [value]="passCertificateNumber">
             </data-row>
 
-            <data-row [label]="'Route number'" [value]="routeNumber"></data-row>
+            <data-row
+                    *ngIf="category != 'ADI3'"
+                    [label]="'Route number'" [value]="routeNumber"></data-row>
 
             <data-row
                     *ngIf="shouldDisplayTestConductedOn()"
                     [label]="'Test conducted on'"
                     [value]="testConductedOn">
             </data-row>
-
-            <data-row [label]="'Independent driving'" [value]="independentDriving"></data-row>
-
-            <data-row [label]="'Candidate appearance true likeness to photo'" [value]="trueLikenessToPhoto ? 'Yes' : 'No'"></data-row>
-
-            <data-row [label]="'Distinguishing features'" [value]="candidateDescription"></data-row>
+            <data-row
+                    *ngIf="category != 'ADI3'"
```

Comment:
Can we create an isADI3 function and use here instead and sort the formatting of tags please

```
 <data-row
       *ngIf="!isADI3()"
       [label]="'Candidate appearance true likeness to photo'" 
       [value]="trueLikenessToPhoto ? 'Yes' : 'No'">
</data-row>
```